awd/helpers.py

```diff
@@ -368,6 +369,16 @@ def valid_sqs_message(self, sqs_message: MessageTypeDef) -> bool:
         return valid
 
 
+def filter_log_stream(stream: StringIO) -> str:
+    """Filter log stream to only ERROR messages for stakeholder email.
+
+    Args:
+        stream: A log stream used to generate an attachment for the stakeholder email.
+    """
+    stream.seek(0)
+    return "".join([line for line in stream if "ERROR" in line])
```

Comment:
I'm betting this works, and strikes me as an elegant approach, but I'm curious if `ERROR` may show up in lines from the traceback?  Is it possible to use `.startswith("ERROR")`?  or maybe it doesn't start with that?  Just a question.

Reply:
It doesn't start with that but I'm going to shift the message formatting so this can use `.startswith("ERROR")`, good call